sdk-setup/src/sdk-assistant

```diff
@@ -328,6 +334,10 @@ if [[ $OPT_ALL && ! $OPT_REGISTER ]]; then
     fail "The '--all' option can only be used with the 'register' command"
 fi
 
+if [[ $OPT_ALL && ! $OPT_REMOVE ]]; then
```

Comment:
Shouldn't the $OPT_ALL be $OPT_SNAPSHOTS_OF?

Reply:
yup, fixed